docs/model_library/operational_water_management/index.rst

```diff
@@ -788,18 +824,20 @@ Completions reuse water is all water that meets completions pad demand, excludin
 
 
 
-**Piping Cost:** :math:`\forall (l,l) \in {PPA,\ldots ,CCA}, t \in T`
+**Piping Cost:** :math:`\forall \textcolor{blue}{l \in (L - O - K)}, \forall \textcolor{blue}{\tilde{l} \in (L - F)}, \forall \textcolor{blue}{(l,\tilde{l}) \in LLA}, \textcolor{blue}{t \in T}`
```

Comment:
```suggestion
**Piping Cost:** :math:`\forall l \in (L - O - K), \forall \tilde{l} \in (L - F), \forall (l,\tilde{l}) \in LLA, t \in T`
```
I am guessing this got copied from the strategic model documentation, but I recommend to remove the blue text here to be consistent with the rest of the operational model documentation.